lab_3_keywords_textrank/main.py

```diff
@@ -385,7 +455,11 @@ def add_edge(self, vertex1: int, vertex2: int) -> int:
                 0 if edge was added successfully, otherwise -1
         In case of vertex1 being equal to vertex2, -1 is returned as loops are prohibited
         """
-        pass
+        if vertex1 == vertex2:
+            return -1
+        self._edges[vertex1] = self._edges.get(vertex1, []) + [vertex2]
```

Comment:
what if these two vertices are already connected?

Reply:
the vertices will be added again, and they will be repeated in the value(
i'll try to fix it now!

Reply:
Exactly, and this is not the behaviour we want

Reply:
done